domino_cost/cost_dashboard.py

```diff
@@ -219,33 +246,70 @@
             ]
         ),
         html.H4("Workload Cost Details", style={"margin-top": "50px"}),
-        html.Div(id="table-container"),
+        html.Div(id="workload-cost-table-container"),
         dbc.Row([], style={"margin-top": "50px"}),
     ],
     className="container",
 )
 
 
+@app.callback(
+    Output("time_span_select", "value"), Input("date-picker-range", "start_date"), Input("date-picker-range", "end_date")
+)
+def update_output(start_date, end_date):
+    if start_date and end_date:
+        return format_date(start_date) + "," + format_date(end_date)
+    else:
+        return "30d"
+
+
+@app.callback(Output(component_id="cloud-cost-card", component_property="style"), [Input("time_span_select", "value")])
+def show_hide_element(time_span):
+    cloud_cost_sum = get_cloud_cost_sum(time_span, base_url=cost_url, headers=auth_header)
```

Comment:
Is there a way to obtain the value from `display_cost_col`?

Reply:
Ah, it was a question related to a [previous comment](https://github.com/dominodatalab/costs-dashboard/pull/7#discussion_r1744230465). But I see that you now have a global variable `cloud_cost_available`. Then maybe we can use it here as well?